pycqed/measurement/calibration/two_qubit_gates.py

```diff
@@ -1099,45 +1104,72 @@ def guess_label(self, **kw):
                     self.label += f"{qb_name}"
 
     def dynamic_phase_block(self, sweep_points, op_code, qubits_to_measure,
-                            prepend_pulse_dicts=None, **kw):
+                            qubits_to_drive=None, prepend_pulse_dicts=None,
+                            num_cz_gates=1, init_for_swap=False, **kw):
         """
         This function creates the blocks for a single DynamicPhase measurement
         task.
         :param sweep_points: sweep points
         :param op_code: (str) the op_code of the flux pulse
         :param qubits_to_measure: (list of str) the qubits on which the
             dynamic phase should be measured (Ramsey qubits)
+        :param qubits_to_drive: (list of str) the qubits on which the
+            initial rotation shall be applied (default=None, in which case
+            qubits_to_measure will be used)
         :param prepend_pulse_dicts: (dict) prepended pulses, see
             prepend_pulses_block
+        :param num_cz_gates: number of sequential CZ gates, default: 1
+        :param init_for_swap: (bool) When the flux pulse is off, initialize the
+            qubits that are not part of qubits_to_drive (for swap gate
+            characterization)
```

Comment:
- The flag is not global, but task-specific. It makes sense that it is task specific because each gate to be characterized corresponds to a task, and whether or not we want this flag depends on the gate type.
- The multitasking framework in principle allows parallel scheduling of tasks with different values of this parameter (even though I am not sure whether we would ever do this).
- qubits_to_measure allows to put multiple qubits into one task because multiple qubits can be affected by the same CZ gate.